docs/install/Knative-with-AKS.md

```diff
@@ -12,7 +12,7 @@ You can find [guides for other platforms here](./README.md).
 
 ## Before you begin
 
-Knative requires a Kubernetes cluster v1.11 or newer. `kubectl` v1.10 is also
+Knative requires a Kubernetes cluster v1.14 or newer. `kubectl` v1.10 is also
```

Comment:
From this kubectl docs:
```
You must use a kubectl version that is within one minor version difference of your cluster. For example, a v1.2 client should work with v1.1, v1.2, and v1.3 master. Using the latest version of kubectl helps avoid unforeseen issues.
```

Looks like this should be v1.13

Reply:
Hm, if it's saying that v1.2 works with v1.1 and v1.3, then somebody using kubectl v1.1 is still good to go with k8s v1.1x, no matter what `x` is, right?

Reply:
I think Dan's right, should we just remove this wording though?